### ISSUE TITLE
Broken custom delimiters when combined with { } at latest dev 26 version

### ISSUE THREAD

**Issue by juangacovas** [smarty-php/smarty#394]
Say I want to use old style delimiters instead of { and } which is nice for us (avoiding literals and stuff)
```
$smarty->left_delimiter = '<%';
$smarty->right_delimiter = '%>';
```
This will produce the following error as soon you try to compile any simple template using the custom delimiters AND anything that also contains { and } 
```
Call to undefined method Smarty_Internal_Templatelexer::yy_r1_()
Line 392 at file: smarty\libs\sysplugins\smarty_internal_templatelexer.php
```
Example non working tpl (setting custom delimiters):
```
<%$test|default:'Default test string'%>
<style>
table.mytable td {
  font-family:Arial;
}
</style>
```
Example working tpl (without { and })
```
<%$test|default:'Default test string'%>
```

I am positive everything was working fine at dev-22 (now dev-26)

Thanks!

**Comment by juangacovas** [comment]
Checked with dev 26 version, still broken, we're unable to use custom delimiters as described.

**Comment by uwetews** [comment]
This is now fixed with version 3.1.32-dev-27
